### PR TITLE
[#5830] fix(client): add error handling for no tag in cli.

### PR DESCRIPTION
**Pull request by LindaSummer**
### What changes were proposed in this pull request?

Add error handle with a friendly output for no tags command line.

### Why are the changes needed?

Now it throws an exception and may make customer confused.

Fix: #5830 

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Unit tests have been attached.

### CODE REVIEW

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/CreateTag.java [hunk 1]
```diff
@@ -52,11 +52,17 @@ public CreateTag(
...
+    switch (tagCount) {
+      case 0:
+        System.err.println(ErrorMessages.TAG_EMPTY);
+        break;
+      case 1:
+        handleOnlyOneTag();
+        break;
+      default:
+        handleMultipleTags();
+        break;
```
Comment (justinmclean) [inline clients/cli/src/main/java/org/apache/gravitino/cli/commands/CreateTag.java:65]: I think an if/else block would be more readable here.

Comment (justinmclean) [reply clients/cli/src/main/java/org/apache/gravitino/cli/commands/CreateTag.java:65]: As these are just tests you can add this to them `@SuppressWarnings("DefaultCharset")`

Comment (tengqm) [reply clients/cli/src/main/java/org/apache/gravitino/cli/commands/CreateTag.java:65]: I agree. A If/Else block is more appropriate in this case.

**Review by justinmclean — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**